src/components/ha-filter-labels.ts

```diff
@@ -44,6 +49,24 @@ export class HaFilterLabels extends SubscribeMixin(LitElement) {
     ];
   }
 
+  private _filteredLabels = memoizeOne(
+    (labels: LabelRegistryEntry[], filter: string | undefined, _value) =>
```

Comment:
`_value` is not used. May as well remove it

Reply:
It's a kind of insider trick to recalculate the memoizeOne when the selection has changed.

Reply:
Please add a comment or someone will delete it in the future

Reply:
```suggestion
    // `_value` used to recalculate the memoization when the selection changes
    (labels: LabelRegistryEntry[], filter: string | undefined, _value) =>
```